---
version: 1
interactions:
- request:
    body: |
      {"account_id":"Test-TestAWSIntegrationCreateConflictErrors-1626196217","account_specific_namespace_rules":{"opsworks":true},"excluded_regions":["us-east-1","us-west-1"],"filter_tags":["testTag","test:Tag2"],"host_tags":["filter:one","filtertwo"],"role_name":"DatadogAWSIntegrationRole"}
    form: {}
    headers:
      Accept:
      - application/json
      Content-Type:
      - application/json
      Dd-Operation-Id:
      - CreateAWSAccount
      User-Agent:
      - datadog-api-client-go/1.2.0 (go go1.16.3; os darwin; arch amd64)
      X-Datadog-Parent-Id:
      - "660878576164921090"
      X-Datadog-Sampling-Priority:
      - "1"
      X-Datadog-Trace-Id:
      - "5320782917602482946"
    url: https://api.datadoghq.com/api/v1/integration/aws
    method: POST
  response:
    body: '{"errors": ["This AWS account ID already exists."]}'
    headers:
      Cache-Control:
      - no-cache
      Connection:
      - keep-alive
      Content-Length:
      - "51"
      Content-Type:
      - application/json
      Date:
      - Tue, 13 Jul 2021 17:10:18 GMT
      Pragma:
      - no-cache
      Strict-Transport-Security:
      - max-age=15724800;
      X-Content-Type-Options:
      - nosniff
      X-Frame-Options:
      - SAMEORIGIN
    status: 409 Conflict
    code: 409
    duration: ""
